Registration forms sub-chapter (#3160)
alan-turing-institute/the-turing-way: book/website/collaboration/organising-conference/reg-form-template.md

- [ ] EPSRC (Engineering and Physical Sciences Research Council)
- [ ] ESRC (Economic and Social Research Council)
- [ ] MRC (Medical Research Council)
- [ ] NERC (Natural Environment Research Council)
- [ ] STFC  (Science and Technology Funding Council)
- [ ] ERC (European Research Council)
- [ ] My Institution/Organisation
- [ ] Wellcome Trust
- [ ] Other [Please add other primary funder.]

23. Why do you want to attend this event? 
*This could be left as an open question or have tick box suggestions if you want to know something more specific*
- [ ] Relevant topic to my research/work
- [ ] To hear talk from specific speaker/s
- [ ] Professional development
- [ ] Academic interest
- [ ] I am part of this project/organisation/community

24. What other themes or topics would you be interested in attending?

25. How did you hear about our event? (can be an open question or with tick boxes)
- [ ] Website
- [ ] Newsletter
- [ ] Twitter (add Twitter handle)
- [ ] LinkedIn (add LinkedIn account link)
- [ ] In another of our events 
- [ ] Direct email
- [ ] Email from a mailing list
- [ ] Email from a colleague
- [ ] I am staff member

> - [ ] Other [Please specify how you first heard about CW23.]

Author: ```suggestion
- [ ] Other [Please specify how you first heard about the event.]
```

> # Template for conference or event registration form

Reviewer: ```suggestion
# Template for Conference or Event Registration Form
```

This template includes all the different questions you might want to include on your registration form. 
This does not mean you have include all of these questions for your event.

Please consider carefully what information you need to collect for your specific event and for reporting and monitoring purposes. 
Therefore, this document is intended to be modified to make it specific for your own purpose. 

```
**Title:** *add title of event*

**Description:** *add a short description of the event and include date, timings with timezone, location, online/in person only/hybrid.*

*Example text*
*We are convening a conference on ____ , which aims to bring together ________ to _______. This will include __________.*  

> *This event is invite only/by application/has a maximum of ___ attendees.  The event is in person/online/hyrbid at _____ on [add date here] between [add times] UTC.*

Reviewer: ```suggestion
*This event is invite only/by application/has a maximum of ___ attendees.  The event is in person/online/hybrid at _____ on [add date here] between [add times] UTC.*
```

*We will be sending out more information about the content and focus of the event shortly.* 

*Please register below to indicate that you will be attending.* 

*Data privacy*

> *No personal data/personal data is collected from you by this form but if you want to read our privacy notice for events at _______, please find it here: [add link to privacy notice]*

Reviewer: ```suggestion
*No personal data/personal data is collected from you by this form but if you want to read our privacy notice for events at _______, please find it here: [add link to privacy notice]* or *We collect personal data about individuals registering for this event. This data includes your name, your email address, your affiliation, country and job title. We will retain this registration information in our server for a year after the completion of the event.*
```

 
## Questions
### Questions to gain permissions

- Do you agree to our privacy notice?
 - [ ] Yes
 - [ ] No

- Are you happy for us to publish audio/visual material of the general event that you may be present in? We will use them for publicity for future events and reporting on this event.
 - [ ] Yes
 - [ ] No

- To make this event fair and inclusive to everyone, you will need to read, agree to and abide by our Participation Guidelines/code of conduct/conference charter (add link to details of document)
 - [ ] Yes
 - [ ] No

- Would you be willing to share your contact name and email with other conference attendees for networking purposes? (Select one) [compulsory question for all type of participants]
 - [ ] Yes, I am comfortable sharing my contact information
 - [ ] No, I prefer to keep my contact information private

### Questions to collect attendees details for running the event (all information collected as open short answers)

> - Name (first name, family name)

Reviewer: ```suggestion
- Name (first name, family name/surname)
```

- Title 

- Preferred pronouns

> - Affiliation/Home institution/Organisation

Reviewer: ```suggestion
- Affiliation/Home Institution/Organisation
```

- Email address

- Dietary requirements *(in person events only)*

- Do you have any disability or accessibility requirements (for remote and/or in-person participation)?
 - [ ] Yes [Please describe your disability or accessibility requirements so that we can support your remote and/or in-person participation at our event]
 - [ ] No

- Registration type *(if there is a choice)*
 - [ ] Online
 - [ ] In person

### Questions for reporting and monitoring purposes 
- Job title

- Career stage (Phases based on https://www.leru.org/files/UK-England.pdf)
 - [ ] Phase 1 (for example: PhD candidate, Research/Teaching Assistant, Junior Research Software Engineer)
 - [ ] Phase 2 (for example: Research Associate/Fellow, Lecturer, Research Software Engineer or Senior Research Software Engineer, Project Manager)
 - [ ] Phase 3 (for example: Senior Lecturer, Reader, Senior Research Fellow, Research Group Leader, Research Software Group Leader, Chief Technical Officer, Programme Manager)
 - [ ] Phase 4 (for example: Full Professor, Director of Research Computing, EU Project Coordinator, Director of Programme)

- What is your research domain/field or area of work?

> - What is the closest JACS 3.0 Principal subject code (https://www.hesa.ac.uk/support/documentation/jacs) that clasifies your research area?

Reviewer: ```suggestion
- What is the closest JACS 3.0 Principal subject code (https://www.hesa.ac.uk/support/documentation/jacs) that classifies your research area?
```

 - [ ] In another of our events 
 - [ ] Direct email
 - [ ] Email from a mailing list
 - [ ] Email from a colleague
 - [ ] I am staff member
 - [ ] Other [Please specify how you first heard about the event.]

- How many of our events have you attended?
 - [ ] 1
 - [ ] 2
 - [ ] 3
 - [ ] 4
 - [ ] 5 or more

- What do you hope to learn and/or gain by attending this event/conference/meeting? (Write in “N/A” or “None” if not applicable.)

### Other conference organisation questions
- Are you presenting at the conference? (Please select one):
 - [ ] Yes, as a speaker
 - [ ] Yes, a poster 
 - [ ] No

- Do you require overnight accommodation?
 - [ ] Yes - I will book this myself
 - [ ] Yes - I need help to book my accommodation?
 - [ ] No

- Do you want to attend out conference dinner that will at [add venue] on [add date] at [add time]?
 - [ ] Yes
 - [ ] No

- Would you like to apply for the conference childcare fund? *Provide details of the childcare fund including who can apply, how much and what details are needed to apply*
 - [ ] Yes, I wish to apply for the childcare fund to support my attendance at the conference - please specify [text box]
 - [ ] No

> - Would you like to apply for the conference’s accessibility fund? *Add details such as - The accessibility fund is designed to support attendees who need additional assistance to participate in the Conference. We can provide financial assistance (up to add amount/person) to help cover support costs for the duration of the conference, or for additional to support remote participation. Please select one of the following options: *

Reviewer: ```suggestion
- Would you like to apply for the conference’s accessibility fund? *Add details such as - The accessibility fund is designed to support attendees who need additional assistance to participate in the conference. We can provide financial assistance (up to add amount/person) to help cover support costs for the duration of the conference, or for additional to support remote participation. Please select one of the following options: *
```
